images/common/scripts/install_gdrcopy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
https://github.com/NVIDIA/gdrcopy?tab=readme-ov-file#deb-package

Reply:
It requires a driver package to be installed. But we can't do this in K8s, because this driver is propagated by nvidia GPU operator

Reply:
Same as from source: ```Compiling the gdrdrv driver requires the NVIDIA driver source code, which is typically installed at /usr/src/nvidia-<version>. ```  https://github.com/NVIDIA/gdrcopy?tab=readme-ov-file#notes

Reply:
This is about compiling the gdrdrv driver and that it needs source codes. But we don't compile the driver, it's propagated by GPU operator. We install gdrcopy libraries & test executables.
Deb packages for these libs & exes have a dependency on the package with gdrdrv driver, that we can't install inside k8s containers.

Reply:
So, I tried to install it as you suggested, but it doesn't work.

Reply:
This issue is about docker, not K8s, but anyway: https://github.com/NVIDIA/gdrcopy/issues/197